frontend/src/metabase/actions/containers/ActionCreatorModal/ActionCreatorModal.unit.spec.tsx

```diff
@@ -68,9 +76,7 @@ async function setup({
     },
   );
 
-  await waitForElementToBeRemoved(() => screen.queryAllByText(/Loading/i));
```

Comment:
Why did you decide to remove the check here and put it in each test instead?

Reply:
This check is `initialRoute`-specific, i.e. it should not be there when the route is `/model/:slug/detail/actions`.
New tests use that route as `initialRoute`.